tests/testCoadds.py

```diff
@@ -130,6 +132,32 @@ def tearDown(self):
         del self.detectTask
         del self.butler
 
+    def getCalexpIds(self, butler):
+        return butler.mapper.index['calexp']['visit'][None]
+
+    def addMaskPlanes(self):
+        butler = lsst.daf.persistence.Butler(DATAREPO_ROOT)
+        # Get the dataId for each calexp in the repository
+        calexpDataIds = self.getCalexpIds(butler)
+        # Loop over each of the calexp and add the CROSSTALK and NOT_DEBLENDED mask planes
+        for ID in calexpDataIds:
```

Comment:
No good reason except you commonly see Id capitalized in general text. I can change it though